packages/datadog-plugin-aerospike/src/index.js

```diff
@@ -0,0 +1,93 @@
+'use strict'
+
+const { storage } = require('../../datadog-core')
+const DatabasePlugin = require('../../dd-trace/src/plugins/database')
+
+class AerospikePlugin extends DatabasePlugin {
+  static get id () { return 'aerospike' }
+  static get operation () { return 'command' }
+  static get system () { return 'aerospike' }
+
+  bindStart (ctx) {
+    const { commandName, commandArgs } = ctx
+    const resourceName = commandName.slice(0, commandName.indexOf('Command'))
+    const store = storage.getStore()
+    const childOf = store ? store.span : null
+    let ns, set, key, bin, index
+    let meta = {}
+
+    if (resourceName.includes('Index')) {
+      [ns, set, bin, index] = commandArgs
+      meta = getMetaForIndex(ns, set, bin, index)
+    } else if (resourceName === 'Query') {
```

Comment:
Commands don't usually start uppercase for other plugins. Does this match how it's done in .NET?

Reply:
yes .NET uses upper cases

Reply:
see here: https://github.com/datadog/dd-trace-dotnet/blob/6aab5e1b02bec9c9b68a33cd06cc9e7a774f14de/tracer/test/snapshots/AerospikeTests.SchemaV0.verified.txt#L1